packages/manager/apps/vrack-services/src/pages/overview/index.tsx

```diff
@@ -48,11 +49,10 @@ export const OverviewTab: React.FC = () => {
   } = useUpdateVrackServices({ key: id });
 
   React.useEffect(() => {
-    tracking.trackPage({
-      name: 'vrack-services::dashboard',
-      level2: '0',
-    });
-  }, []);
+    if (!isLoading) {
```

Comment:
Why you have to check for loading ? because without it calls multiple times with the polling ? In that cas maybe we can do without useEffect (maybe with useCallback) ?

Reply:
Because the overview page is really displayed if the fetch loads, if it sends an error because the id is not good, we display error page so we should not send a tracking for overview page I suppose

Reply:
If isLoading change only once then it is ok

Reply:
Yes it will probably cause problem if we update the vrack services name